tests/integration-e2e/test_cli_opts.py

```diff
@@ -19,20 +19,23 @@ def test_save_restore_cli():
     for simulator in ("NEURON", "CORENEURON"):
         test_folder = tempfile.TemporaryDirectory("cli-test-" + simulator)  # auto removed
         test_folder_path = Path(test_folder.name)
-        for action, tstop in (("save", 100), ("restore", 200)):
+        for actions, output_dir, tstop in (([("save", "output_p1/checkpoint")], "output_p1", 100),
```

Comment:
Nitpick: As the structure being looped became more complex it might be worth defining outside to help readability.

Reply:
I have separated the test parameters. Please have a look if it is clearer now.